keps/0000-kep-template.md

```diff
@@ -139,14 +151,74 @@ What are the risks of this proposal and how do we mitigate.
 Think broadly.
 For example, consider both security and how this will impact the larger kubernetes ecosystem.
 
-## Graduation Criteria
+How will security be reviewed and by whom?
+How will UX be reviewed and by whom?
+
+Consider including folks that also work outside the sig or subproject.
+
+## Design Details
+
+### Test Plan
+
+**Note:** *Section not required until targeted at a release.*
+
+What is the test plan for the component?  E2e, integration, unit testing.  How will it be tested in isolation vs
+with other components?
+
+No need to outline all of the test cases, just the general strategy.
```

Comment:
While all test cases don't need to be covered, anything that would count as tricky in the implementation and anything particularly challenging to test should be called out. 

We also need to emphasize that all code is expected to have adequate tests (eventually with coverage expectations). A link to the testing conventions may be useful:
https://github.com/kubernetes/community/blob/master/contributors/devel/testing.md